docker-run-generate-outputs.sh

```diff
@@ -46,6 +46,8 @@ OUTPUT_CODED_DIR=$9
 OUTPUT_MESSAGES_CSV=${10}
 OUTPUT_INDIVIDUALS_CSV=${11}
 OUTPUT_PRODUCTION_CSV=${12}
+OUTPUT_AUTO_CODING_TRACED_JSONL=${13}
+GENERATE_ANALYSIS_FILES=${14}
```

Comment:
I'm not sure about the order of these. I think I would put generate_analysis_files before the file paths, and possible re-order the auto_coding_traced_data_jsonl to be before the analysis outputs, in order to better match pipeline order.